cli/start.go

```diff
@@ -189,6 +189,9 @@ func runStart(_ *cobra.Command, _ []string) error {
 	}
 
 	info := build.GetInfo()
+	// We log build information to stdout (for the short summary), but also
+	// to stderr to coincide with the full logs.
+	log.Infof("[build] %s @ %s (%s)", info.Tag, info.Time, info.GoVersion)
```

Comment:
is `log.Infof("%s", info.Short())` too spammy? If not, I have a slight preference for it.